core/shared/src/main/scala/zio/FiberFailure.scala

```diff
@@ -26,11 +27,27 @@ import zio.stacktracer.TracingImplicits.disableAutoTrace
  * This class is used to wrap ZIO failures into something that can be thrown, to
  * better integrate with Scala exception handling.
  */
-final case class FiberFailure(cause: Cause[Any]) extends Throwable(null, null, true, false) {
+final case class FiberFailure(cause: Cause[Any]) extends Throwable(null, null, true, true) {
+
+  private val javaStackTrace: Array[StackTraceElement] = {
```

Comment:
I might be wrong but I don't think this is used at the moment?

Reply:
As preferred of filtering lets use it

Reply:
Yes! we don't need it now